client/sites/v2/sites-list/router.tsx

```diff
@@ -126,3 +69,9 @@ export const getRouter = ( { basePath }: { basePath: string } ) => {
 
 	return router;
 };
+
+export const routerConfig = {
+	basePath: '/',
+};
```

Comment:
I actually like this better where it was. This isn't "router" config, this is "app" config IMO. It should mimic `AppConfig` from the v2 dashboard. We haven't used it a lot yet, but leaf components/routes should be able to access the config and know that whether they're being rendered in v2 or in the backport they can rely on accessing that config data.

Reply:
Makes sense. I removed it to simplify it.